app/routes/Settings/components/Layout.js

```diff
@@ -7,7 +7,7 @@ import styles from './index.css'
 class Layout extends Component {
   render() {
     return (
-      <Page account logOut>
+      <Page account logOut padding='noPadding'>
```

Comment:
if the most common situation now is having the <Page> component without padding consider change the component removing the padding making it optional

Reply:
The most common situation is having the Page component with padding